builds/e2e/connectivity.yaml

```diff
@@ -10,10 +10,18 @@ resources:
   pipelines:
   - pipeline: images
     source: 'Azure-IoT-Edge-Core Build Images'
-    branch: master
+    branch: 'release/1.2'
+    trigger:
+      branches:
+      - master
+      - release/*
   - pipeline: packages
     source: 'Azure-IoT-Edge-Core Edgelet Packages'
-    branch: master
+    branch: 'release/1.2'
+    trigger:
+      branches:
+      - master
+      - release/*
```

Comment:
This is the working configuration used for end to end test. But if I had to guess, only `release/1.2` (and not `master`) is required under branches.

Reply:
I think you're right. But we list both so that it's easier to keep the two branches in sync.